new-frontiers-14/frontier-station-14
# Station Records: Populate job list whenever possible

## About the PR



Populates the job lists and filters for empty stations.  Job listings exclude overflow jobs (e.g. contractor, pilot, mercenary on Frontier Outpost), and print out the proper name of the job.

## Why / Balance


Being able to set station jobs is important.

## How to test


1. Spawn on Frontier Outpost.  Check station records console, all jobs should be listed in their default amounts, with their proper names (Station Traffic Controller, Station Representative), no overflow jobs should be listed (Contractor, Pilot, Mercenary).
2. Warp to NFSD Outpost.  Check station records console, all jobs should be listed in their default amounts, with their proper names (Detective, not DetectiveNF), although no records should be present.
3. Warp to the Pirate Cove.  Check the station records console, all jobs should be listed in their default amounts, with their proper names, although no records should be present.

## Media


A picture of the NFSD Outpost, with no valid records.
![image](https://github.com/user-attachments/assets/43677466-7216-4009-b950-9295775a7d75)

A picture of Frontier Outpost, with no overflow jobs listed.
![image](https://github.com/user-attachments/assets/f0053bd3-b51d-4770-88c3-354574089c25)

## Requirements


## Breaking changes


**Changelog**



:cl:
- fix: The station records computer should now properly list all jobs whenever possible.
